source/common/http/http1/codec_impl.cc

```diff
@@ -742,7 +742,8 @@ StatusOr<ParserStatus> ConnectionImpl::onHeadersComplete() {
   // CONNECT request has no defined semantics, and may be rejected.
   if (request_or_response_headers.TransferEncoding()) {
     const absl::string_view encoding = request_or_response_headers.getTransferEncodingValue();
-    if (!absl::EqualsIgnoreCase(encoding, header_values.TransferEncodingValues.Chunked) ||
+    if (!HeaderUtility::headerValueContainsIgnoreCase(
+            encoding, header_values.TransferEncodingValues.Chunked) ||
```

Comment:
Does the ordering of entries in the transfer encoding header matter?  Your code considers both "unkown,chunked" and "chunked,unknown" as "chunked".  Only the first of these two should be considered "chunked".